src/Apropos/HasPermutationGenerator/Morphism.hs

```diff
@@ -29,8 +29,10 @@ instance Show (Morphism p m) where
 composeMorphisms :: Morphism p m -> Morphism p m -> Morphism p m
 composeMorphisms a b =
   Morphism
-    { name = name a <> name b
-    , match = match a :&&: match b
+    { -- composing morphisms is analogous to >=>
+      name = "(" <> name a <> ") >=> (" <> name b <> ")"
+    , -- TODO is this correct in the case where contract a changes rather match b matches?
+      match = match a :&&: match b
```

Comment:
I'm not sure what this TODO means. Can you clarify?

Reply:
Oh I'll remove that. That was before I understood that this was composing in parallel and running the match first made sense.

Reply:
Coolio :+1: